src/table/table-body.jsx

```diff
@@ -183,10 +183,11 @@ const TableBody = React.createClass({
 
     return React.Children.map(this.props.children, (child) => {
       if (React.isValidElement(child)) {
+        const {selected} = child.props;
         const props = {
           displayRowCheckbox: this.props.displayRowCheckbox,
           hoverable: this.props.showRowHover,
-          selected: this._isRowSelected(rowNumber),
+          selected: selected !== undefined ? selected : this._isRowSelected(rowNumber),
```

Comment:
@ccomb My apologies, I forgot how much hijacking this table component does -- not having tests for it results in this sort of uncertainty!